### PR TITLE
App & apps' changelog on release 

### PR DESCRIPTION
**Pull request by tomahawk28**
Toward giantswarm/giantswarm#8135

App list on AWS only for node pool cluster. 
All changelogs, app versions are updated as the latest today (Feb 5). 

Spec: https://github.com/giantswarm/giantswarm/blob/master/areas/managed-services/specs/app-on-release.md

It will close https://github.com/giantswarm/releases/pull/70

### CODE REVIEW

File: aws.yaml [hunk 1]
```diff
@@ -1,4 +1,35 @@
 - active: true
+  apps:
```
Comment (rossf7) [inline aws.yaml]: This is updating the 11.0.0 release but it's already active. Please add a 11.0.1 WIP release with the new cluster-operator release and apps.

This is easy to miss as the current master doesn't have a WIP for 11.X.

Comment (tomahawk28) [reply aws.yaml]: Should be 11.1.0? since it's not bug fix actually.

Comment (rossf7) [reply aws.yaml]: Let's use patch for now. These changes are about improving our developer experience. This is more interesting for us than for customers.

When this WIP version is released we can upgrade to a minor version if necessary.

File: aws.yaml [hunk 2]
```diff
@@ -1,3 +1,49 @@
...
+    - app: net-exporter
+      version: 1.5.1
+    - app: node-exporter
+      version: 1.2.0
+      componentVersion: 0.18.1
+    - app: cert-manager
+      version: 1.0.4
+      componentVersion: 0.9.0
+    - app: cluster-autoscaler
+      version: 1.1.4
```
Comment (rossf7) [inline aws.yaml]: There is no tarball for this release. https://github.com/giantswarm/default-catalog Can you check the version?

This is very easy to miss and I think we should have an automated check. So we know the versions here actually exist in the catalog.

@puja108 @piontec WDYT? Should we have an issue for that?

File: aws.yaml [hunk 3]
```diff
@@ -1,3 +1,49 @@
...
+      version: 1.2.0
+      componentVersion: 0.5.18
+    - app: kiam
+      version: 1.1.0
+      componentVersion: 3.5.0
+  authorities:
+    - name: app-operator
+      version: 1.0.0
+    - name: aws-operator
+      version: 8.1.0
```
Comment (rossf7) [inline aws.yaml]: This should also be WIP. https://github.com/giantswarm/aws-operator/blob/de013a3681525e3b47525a7e57989955c87ad226/pkg/project/project.go#L4

File: aws.yaml [hunk 4]
```diff
@@ -1,3 +1,49 @@
...
+    - name: cert-operator
+      version: 0.1.0
+    - name: chart-operator
+      version: 0.7.0
```
Comment (rossf7) [inline aws.yaml]: This chart-operator version is legacy cruft. It isn't the version used and I'd actually like to remove it.

@MarcelMue WDYT?

File: aws.yaml [hunk 5]
```diff
@@ -1,3 +1,49 @@
+- active: false
+  apps:
+    - app: chart-operator
+      version: 0.11.4
+    - app: coredns
+      version: 1.1.3
+      componentVersion: 1.6.5
```
Comment (rossf7) [inline aws.yaml]: Let's also order the version fields. I think that is tidier.

File: aws.yaml (hunk 4 above)
Comment (MarcelMue) [reply aws.yaml]: Works for me. No need to keep dead weight.

File: aws.yaml (hunk 2 above)
Comment (tomahawk28) [reply aws.yaml]: It was in a changelog but not released yet. I just publish a new 1.1.4 tag there, Thanks for the notice.

File: aws.yaml (hunk 5 above)
Comment (tomahawk28) [reply aws.yaml]: I put it like deliberately, but let's keep in order as you said.

File: aws.yaml (hunk 2 above)
Comment (rossf7) [reply aws.yaml]: Thanks I'd still like to have the automated check. But I think it can be done after these changes.

**Comment by corest** [comment]
I've probably missed spec discussion, but maybe can get answer here - why do we go with `componentVersion` while charts actually use `appVersion`?

**Comment by rossf7** [comment]
> I've probably missed spec discussion,

@tomahawk28 Can you link the spec in the PR desc?

> why do we go with componentVersion while charts actually use appVersion?

@corest In the API responses we have components and versions. So it aligns with that.

We also decided we wanted to be more opinionated on naming and not always align with Helm. 

For example in the next version of the app CRD `namespace` will probably be renamed to `targetNamespace`.

**Comment by tomahawk28** [comment]
@corest `appVersion` is misleading the context a bit like it's like giantswarm's app version despite the fact that we have `version` for itself.  
Compare to `componentVersion`, it gives a sense that it's the version of the component inside giantswarm's app.

**Comment by corest** [comment]
Got it. Thx for explanation @tomahawk28 @rossf7

**Comment by tomahawk28** [comment]
going to ping @giantswarm/team-ludacris . 
Sorry

**Comment by tomahawk28** [comment]
Going to merge.

**Pull request merged**